readme.md

```diff
@@ -374,10 +374,10 @@ async function main() {
     .use(remarkPresetLintConsistent)
     // Few recommended rules.
     .use(remarkPresetLintRecommended)
-    // `remark-lint-list-item-indent` is configured with `tab-size` in the
```

Comment:
I think this change should be reverted. See: https://github.com/remarkjs/remark-lint/pull/281#issuecomment-1050911561.